app/assets/javascripts/app/discussions/templates/edit_header_body.html.slim

```diff
@@ -0,0 +1,6 @@
+md-button.md-primary.md-hue-1 ui-sref="discussions.new.title"
+  i.fa.fa-chevron-left.u-alpha
```

Comment:
Why is there a `u-alpha` here? If you need a different type a button create a modifier.